### PR TITLE
Update messaging log code to use postgres models

### PR DESCRIPTION
**Pull request by gcapalbo**
Updates the majority of the code that references messaging logs to use the postgres models.

Updates all of the code for:
- CallLog
- ExpectedCallbackEventLog
- LastReadMessage

Updates the majority of the code for:
- SMSLog

The remaining updates for SMSLog will follow in subsequent PRs

@orangejenny

### CODE REVIEW

File: corehq/apps/ivr/models.py [hunk 1]
```diff
@@ -140,3 +140,32 @@ def _migration_get_fields(cls):
...
+            date__gte=from_timestamp,
+            answered=True
+        )
+
+        if to_timestamp:
+            qs = qs.filter(
+                date__lte=to_timestamp
+            )
+
+        return len(qs[:1]) > 0
```
Comment (orangejenny) [inline corehq/apps/ivr/models.py]: isn't this just saying `len(qs) > 1`? seems unnecessarily complex.

**Comment by orangejenny** [comment]
Looks good to me pending tests.

Is there anyone with more context who should take a look? (Is there a second-in-command for the SMS code?)

File: corehq/apps/ivr/models.py (hunk 1 above)
Comment (gcapalbo) [inline corehq/apps/ivr/models.py]: my thinking here was that the slicing for `qs[:1]` forces it to only fetch the first record to determine if it exists or not (`len(qs)` forces it to fetch all records).  but now that i think about it, it's probably just as good to use qs.count() here. will update

Comment (gcapalbo) [inline corehq/apps/ivr/models.py]: (updated)

**Comment by gcapalbo** [comment]
i don't think anyone else has more context, but i'm good with all these changes (ran the manual sms and ivr test suites and they both pass)

**Comment by czue** [comment]
:boom:

**Pull request merged**